inference/core/interfaces/stream/stream.py

```diff
@@ -70,6 +70,7 @@ def __init__(
         on_prediction: Callable = None,
         on_start: Callable = None,
         on_stop: Callable = None,
+        simulate_process_delay: float = 0,
```

Comment:
Is this just for testing? If so, should we remove it?

Reply:
@sberan maybe you have an un-pushed commit? Or maybe just missed this piece of the debugging code when removing? :)